feat(convert) : add script allow to convert markdown to ascidoc (#113)
bonitasoft/bonitasoft.github.io: docs/migrate-from-old-documentation-site/doc-content-conversion-for-new-doc-site.adoc

For such a page, the html part must be commented prior running conversion otherwise the process fails.
====

> Thanks to the scripts link:../scripts/convert_md_to_asciidoc.py[convert_md_to_asciidoc.py], the conversion was done automatically.

Reviewer: ```suggestion
The script link:../scripts/convert_md_to_asciidoc.py[convert_md_to_asciidoc.py] performs the conversion automatically.
```

> From the root old documentation  folder, run the following command

Reviewer: ```suggestion
From the root of the old documentation folder, run the following command
```

[source,bash]
----
 python convert_md_to_asciidoc.py
----

.The script will :
[info]

> * Sanitize the markdown source file to render it more easy to be convertible.

Reviewer: ```suggestion
* Sanitize the markdown source files to make theme convertible (update syntax). 
```
